src/js/main.js

```diff
@@ -612,25 +613,44 @@
 
                 if (isWinning()) {
                     drawInfoText(cx, "YOU WIN!");
-                    stopTune(); // Winning effects here?        `
-                } else if (((performance.now() - levelStartTime) < HELP_TEXT_DISPLAY_TIME) && currentMap.text) {
+                    playTune("end"); // Winning effects here? 
+                  } else if (player.dead) {
+                    drawInfoText(cx, "GAME OVER!");
+                    playTune("end"); // End effects here?  `
+                  } else if (((performance.now() - levelStartTime) < HELP_TEXT_DISPLAY_TIME) && currentMap.text) {
                     drawInfoText(cx, currentMap.text);
                 }
             }
         });
     }
 
-    function playTune() {
-        mainTune.play();
-    }
-    function stopTune() {
-        mainTune.pause();
-        mainTune.currentTime = 0.0;
+    function playTune(tune) {
+        switch(tune) {
+            case "main": {
+                mainTune.currentTime = 0;
+                var promise = mainTune.play();
+                if (promise !== undefined) {
```

Comment:
Would it be more robust to check just by if (promise), that would handle the null case as well?